### ISSUE TITLE
User ID seems to be case-sensitive

### ISSUE THREAD

**Issue by webberian** [zenphoto/zenphoto#1394]
The user ID I created starts with a lowercase letter, e.g. "superuser". When I try to log in with an auto-capitalized user name on my phone - e.g. "Superuser" - login is refused. However, when I manually change the user ID to lowercase it works.
I'm using the latest version 1.6.

It would be most logical and user-friendly if usernames were not case-sensitive when logging in.

**Comment by acrylian** [comment]
Yes, the user name itself is case sensitive, the internal check for taken user names is not. We'll think about if we change this.
